### PR TITLE
fix generator to be consistent with current plugin system

### PR DESCRIPTION
**Pull request by akurinnoy**
This PR updates the plugin generator to be consistent to https://github.com/theia-demo-plugins/wiptheia/pull/1

Signed-off-by: Oleksii Kurinnyi <okurinny@redhat.com>

### CODE REVIEW

File: templates/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export function doStartThings() {
     );
 }
 
-export function doStopThings(api: typeof theia) {
+export function stop(api: typeof theia) {
```
Comment (benoitf) [inline templates/index.ts:19]: I think we are not sending as well theia object on stop (I forget to remove it)
it can be fixed later as well

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
do we need CI on this project as well ?

**Comment by akurinnoy** [comment]
I think, yes

File: src/app/index.ts [hunk 2]
```diff
@@ -93,6 +100,15 @@ module.exports = class TheiaPlugin extends Base {
             (this.options as any).pluginName = answers.name
         }
 
+        if (!(this.options as any).publisher) {
```
Comment (benoitf) [inline src/app/index.ts]: what happens if here is no publisher ? (for example I only want to create a plugin locally) ? is there a default publisher value for that case ?

Comment (akurinnoy) [reply src/app/index.ts]: You can omit publisher ID and it will equal an empty string.

Comment (benoitf) [reply src/app/index.ts]: should publisher be then only a "parameter" of the generator (and not a question to user) ?
(like for theia extension where you can specify license, etc) so we don't ask the publisher information at first

it makes the flow to create the plugin as fast as possible.
We can check that publisher is valid only when we want to publish the extension

Comment (akurinnoy) [reply src/app/index.ts]: For now, it is better to have a parameter.
And later it would be stored and read from `.*rc` file as default value.

Comment (benoitf) [reply src/app/index.ts]: ok so we remove the question but we let "-p" option to give publisher and later we'll implement `*.rc`stuff ?

Comment (akurinnoy) [reply src/app/index.ts]: yes, 
I will add a fix

**Review by benoitf — APPROVED** [review]

**Pull request merged**